midenc-compile/src/stages/parse.rs

```diff
@@ -14,6 +20,34 @@ use midenc_session::{
 
 use super::*;
 
+/// A wrapper that emits WebAssembly text format (WAT).
```

Comment:
*Nit:* Probably better to define this in the `midenc-frontend-wasm` crate, and it'll reduce the clutter in this file as an extra benefit.

Reply:
Done.